hugegraph-api/src/main/java/com/baidu/hugegraph/api/graph/EdgeAPI.java

```diff
@@ -157,6 +159,66 @@ public String create(@Context GraphManager manager,
         });
     }
 
+    /**
+     * Batch update steps are same like vertices
+     **/
+    @PUT
+    @Timed
+    @Decompress
+    @Path("batch")
+    @Consumes(APPLICATION_JSON)
+    @Produces(APPLICATION_JSON_WITH_CHARSET)
+    public String update(@Context HugeConfig config,
+                         @Context GraphManager manager,
+                         @PathParam("graph") String graph,
+                         BatchEdgeRequest req) {
+        LOG.debug("Graph [{}] update edges: {}", graph, req.jsonEdges);
+        checkUpdatingBody(req.jsonEdges);
+        checkBatchSize(config, req.jsonEdges);
+        // Not support createIfNotExist equals false now
+        E.checkArgument(req.createIfNotExist == false,
+                        "Param 'create_if_not_exist' is not supported now");
```

Comment:
Param => Parameter
also check req != null at the first line
and check other parameters in req.
we can move all parameters check(include `req.createIfNotExist`) of BatchEdgeRequest to BatchEdgeRequest.checkUpdate() method.